provision/acc_provision/cloud_provision.py

```diff
@@ -169,12 +169,13 @@ def Run(self, flavor_opts, kube_yaml_gen_func):
         output_tar = self.args.output_tar
         operator_cr_output_file = self.args.aci_operator_cr
         gen(self.config, output_file, output_tar, operator_cr_output_file)
-        if self.args.flavor == "aks":
-            self.print_aks_setup()
-        elif self.args.flavor == "eks":
-            self.print_eks_setup()
-        else:
-            self.print_openshift_setup()
+        if "test_run" not in self.args._asdict():
```

Comment:
Per my earlier request, was this tested with the standalone acc-provision tool (when its not being invoked from the tests)? I think this is still going to create a problem when invoked from the acc-provision tool. While "args" is a namedtuple in the tests, I don't think its a namedtuple when the acc-provision tool runs (and if not, it won't have the _asdict() method). You will need to track the "args" definition in the main().